ansible/host_vars/wuerfel

```diff
@@ -0,0 +1 @@
+sshd_allowgroup: ssh
```

Comment:
I would call the variable(s) `sshd_allowgroups_host` and `sshd_allowgroups_group`. As with `sshd_allowusers_*` the first will be set using host host_vars and the latter using group_vars.